tests/integration_test.go

```diff
@@ -0,0 +1,90 @@
+package tests
+
+import (
+	"bytes"
+	"io/ioutil"
+	"path"
+	"path/filepath"
+	"testing"
+
+	skynet "github.com/NebulousLabs/go-skynet"
+	"gopkg.in/h2non/gock.v1"
+)
+
+// TestUploadAndDownloadFile tests uploading and downloading a single file.
+func TestUploadAndDownloadFile(t *testing.T) {
```

Comment:
I'm not sure how these tests validate the inner workings of the library logic itself. Perhaps I don't fully understand but it looks to me it only verifies the most basic case and ignores the data that's being sent, request headers, query string params etc?

For example the `srcFile`, does that file exist? Are we actually mocking the post body? I think `gock` is capable of all of this.